tests/rt_gnu.p7.conf

```diff
@@ -0,0 +1,22 @@
+###################################################################################################################################################################
```

Comment:
Should this file be deleted before merging?

Reply:
yes, both of the p7.conf files will be deleted.